app/modules/listings/components/listing_element_show.svelte

```diff
@@ -0,0 +1,125 @@
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte'
+  import { isNonEmptyPlainObject } from '#app/lib/boolean_tests'
+  import Flash from '#app/lib/components/flash.svelte'
+  import { icon } from '#app/lib/icons'
+  import EntityImage from '#entities/components/entity_image.svelte'
+  import AuthorsInfo from '#entities/components/layouts/authors_info.svelte'
+  import Ebooks from '#entities/components/layouts/ebooks.svelte'
+  import EntityTitle from '#entities/components/layouts/entity_title.svelte'
+  import Infobox from '#entities/components/layouts/infobox.svelte'
+  import Summary from '#entities/components/layouts/summary.svelte'
+  import ListingElementComment from '#modules/listings/components/listing_element_comment.svelte'
+  import { I18n } from '#user/lib/i18n'
+
+  export let element, entity, isCreatorMainUser
+
+  const dispatch = createEventDispatcher()
+  const { type, claims } = entity
+
+  let flash
+</script>
+
+<div class="listing-element-show">
+  <div class="entity-type-label">
+    {I18n(type)}
+  </div>
+  <EntityTitle
+    {entity}
+    hasLinkTitle={true}
+  />
+  <div class="entity-info-row">
+    {#if isNonEmptyPlainObject(entity.image)}
+      <EntityImage
+        {entity}
+        size={256}
+      />
+    {/if}
+    <div class="entity-infobox">
+      <AuthorsInfo {claims} />
+      <Infobox
+        {claims}
+        entityType={type}
+        shortlistOnly={true}
+      />
```

Comment:
Authors info are displayed twice on works/series. Example here with a list element based on wd:Q8337
![2024-11-12_22-03](https://github.com/user-attachments/assets/511ac36f-f2ce-4a66-8d41-7c4129dadade)

Reply:
done in f942b90f9